### PR TITLE
Fix default ts format

### PR DESCRIPTION
**Pull request by lbudai**

### CODE REVIEW

**Comment by kvch** [comment]
@kira-syslogng retest this please

**Comment by kira-syslogng** [comment]
Build ERROR

**Comment by presidento** [comment]
@kira-syslogng retest this please

**Comment by kira-syslogng** [comment]
Build ERROR

**Comment by kira-syslogng** [comment]
Build ERROR

**Comment by presidento** [comment]
@kira-syslogng retest this please

**Comment by presidento** [comment]
The previous build error seem to be a environmental hiccup, because this test passed the parallel executions.

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: lib/template/templates.c [hunk 1]
```diff
@@ -325,7 +324,7 @@ log_template_options_defaults(LogTemplateOptions *options)
 {
   memset(options, 0, sizeof(LogTemplateOptions));
   options->frac_digits = -1;
-  options->ts_format = TS_FMT_ISO;
+  options->ts_format = -1;
   options->on_error = -1;
```
Comment (bazsi) [inline lib/template/templates.c]: I don't get this patch or its original. I think ts_format needs to be set to -1 here and log_template_options_init() be called to initialize it.

Comment (lbudai) [inline lib/template/templates.c]: This is set back to '-1' (change its value was a mistake...)
Please, approve the changes, until that we cannot merge this.

**Comment by lbudai** [comment]
@bazsi: we need to merge this PR right now as this is blocking us.

**Pull request merged**